shared/jsonreport/jsonreport.go

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
Sorry I didn't make it clear. I think this package should be under flaky-test-reporter (this is fine in go, i.e. https://github.com/knative/test-infra/tree/master/tools/monitoring), as the logic is so close.